### PR TITLE
Add internal/external traffic KIC guide

### PR DESCRIPTION
**Pull request by mheap**
### Description

Add a guide that explains how to run internal/external ingresses using KIC

### Testing instructions

Preview link: /kubernetes-ingress-controller/latest/guides/custom-class/internal-external/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by Guaris** [comment]
@mheap who do you want to review this? Can we tag them in it?

**Comment by mheap** [comment]
@Guaris It needs a writer edit. I’ll take care of the SME edit

**Comment by mheap** [comment]
Just waiting for a copyedit now

File: app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Using Custom Classes to split Internal/External traffic
+---
+
+{{ site.kic_product_name }} automatically creates a `kong` `IngressClass` when installed. All of the example ingress definitions in the documentation set `spec.ingressClassName: kong`, which allows things to work by default.
+
+Advanced users of {{ site.kic_product_name }} may wish to split traffic in to internal and external ingress definitions. This requires multiple {{ site.kic_product_name }} instances, with each pointing to a different `IngressClass`
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md:7]:
```suggestion
Advanced users of {{ site.kic_product_name }} may wish to split traffic in to internal and external ingress definitions. This requires multiple {{ site.kic_product_name }} instances, each pointing to a different `IngressClass`
```

File: app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Understanding IngressClass
+
+The `IngressClass` resource is what binds an `Ingress` definition to an ingress controller. The value in the `spec.controller` field defines which ingress controller will process those ingress definitions. {{ site.kic_product_name }} processes any `IngressClass` where `spec.controller` is set to `ingress-controllers.konghq.com/kong`.
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md:11]:
```suggestion
The `IngressClass` resource binds an `Ingress` definition to an ingress controller. The value in the `spec.controller` field defines which ingress controller will process those ingress definitions. {{ site.kic_product_name }} processes any `IngressClass` where `spec.controller` is set to `ingress-controllers.konghq.com/kong`.
```

File: app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Installing Kong
+
+{{ site.kic_product_name }} processes one `IngressClass` per installation. {{ site.kic_product_name }} requires two deployments to split internal and external traffic.
+
+Each deployment lives in it's own namespace, and the `controller.ingressController.ingressClass` value is set depending on if that deployment should handle internal or external traffic.
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md:17]:
```suggestion
Each deployment lives in its namespace, and the `controller.ingressController.ingressClass` value is set depending on whether that deployment should handle internal or external traffic.
```

**Review by Guaris — APPROVED** [review]
Approving this, committing the copy edit changes and merging it in.

File: app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md (hunk 3 above)
Comment (mheap) [reply app/_src/kubernetes-ingress-controller/guides/custom-class/internal-external.md]: This changes the meaning. It should be `it's own` or words that indicate exclusivity

**Pull request merged**